packages/salesforcedx-vscode-core/src/commands/util/commandLog.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { CommandEvent, CommandEventStream, CommandEventType } from '@salesforce/salesforcedx-utils-vscode';
+import { WorkspaceContext } from '../../context';
+import { CommandLogEntry } from './commandLogEntry';
+
```

Comment:
The stand alone methods...should they be static methods on CommandLog?

Reply:
My goal was to hide the `CommandLog` implementation from the API. If I move a new service to the API, I can get rid of the standalone methods.